recipe/bld.bat

```diff
@@ -1,13 +1,14 @@
 @echo On
 
 if "%cuda_compiler_version%" == "None" (
+    set KATAGO_BACKEND="EIGEN"
     set build_with_cuda=
     set USE_CUDA=0
-    set KATAGO_BACKEND="EIGEN"
 ) else (
+    set KATAGO_BACKEND="CUDA"
     set build_with_cuda=1
+    set USE_CUDA=1
     set desired_cuda=%CUDA_VERSION:~0,-1%.%CUDA_VERSION:~-1,1%
```

Comment:
I think this line is not working as intended because `CUDA_VERSION` is not defined?

See the logs:

```
(%BUILD_PREFIX%) %SRC_DIR%>set "PATH=C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v~0,-1CUDA_VERSION:~-1,1\bin;..." 
```

If you just need the cuda version, you can use the one used by conda build in the matrix, e.g. `%cuda_compiler_version%`. Check the logs (before CMake runs) to check how the env vars are being set.

Reply:
Thanks!

Trying now adding:

```
    set CUDA_VERSION=%cuda_compiler_version%
    set desired_cuda=%CUDA_VERSION:~0,-1%.%CUDA_VERSION:~-1,1%
```